doc/MONSTER_SPECIAL_ATTACKS.md

```diff
@@ -55,218 +74,222 @@ In the case of separately defined attacks the object has to contain at least an
-- ```ABSORB_MEAT``` Absorbs adjacent meat items (maximal absorbable item volume depends on the monster's volume), regenerating health in the process.
+- ```ABSORB_MEAT``` Absorbs adjacent meat items (maximum absorbable item volume depends on the monster's volume), regenerating health in the process.
 - ```ACID``` Spits acid.
 - ```ACID_ACCURATE``` Shoots acid that is accurate at long ranges, but less so up close.
 - ```ACID_BARF``` Barfs corroding, blinding acid.
 - ```ANTQUEEN``` Hatches/grows: `egg > ant > soldier`.
-- ```BIO_OP_BIOJUTSU``` Attack with any of the below martial art attacks.
-- ```BIO_OP_TAKEDOWN``` Takedown attack, bashes either the target's head or torso and inflicts `downed`.
+- ```BIO_OP_BIOJUTSU``` Attacks with any of the below martial art attacks.
 - ```BIO_OP_DISARM``` Disarming attack, does no damage.
 - ```BIO_OP_IMPALE``` Stabbing attack, deals heavy damage and has a chance to cause bleeding.
-- ```BITE``` Bite attack that can cause deep infected wounds. If the attacker is humanoid, the target must be `grabbed` before BITE can trigger.
-- ```BOOMER_GLOW``` Spit glowing bile.
-- ```BOOMER``` Spit bile.
-- ```BRANDISH``` Brandish a knife at the player.
-- ```BREATHE``` Spawns a `breather` - `breather hub` only!
+- ```BIO_OP_TAKEDOWN``` Takedown attack, bashes either the target's head or torso and inflicts `downed`.
+- ```BITE``` Bite attack that can cause deep infected wounds.  If the attacker is humanoid, the target must be `GRAB`bed before `BITE` can trigger.  See also `bite` below.
+- ```BOOMER_GLOW``` Spits glowing bile.
+- ```BOOMER``` Spits bile.
+- ```BRANDISH``` Brandishes a knife at the player.
+- ```BREATHE``` Spawns a `breather`.  Note: `breather hub` only!
 - ```CALLBLOBS``` Calls 2/3 of nearby blobs to defend this monster, and sends 1/3 of nearby blobs after the player.
-- ```CHICKENBOT``` LEGACY - Robot can attack with tazer, M4, or MGL depending on distance.
+- ```CHICKENBOT``` Robot can attack with tazer, M4, or MGL depending on distance.  Note: Legacy special attack.
 - ```COPBOT``` Cop-bot warns then tazes the player.
 - ```DANCE``` Monster dances.
 - ```DARKMAN``` Can cause darkness and wraiths to spawn.
-- ```DERMATIK_GROWTH``` Dermatik larva grows into an adult - obsoleted by them using `age_grow`.
 - ```DERMATIK``` Attempts to lay dermatik eggs in the player.
+- ```DERMATIK_GROWTH``` Dermatik larva grows into an adult.  Note: obsoleted by them using `age_grow`.
```

Comment:
Maybe obsoleted function should not appear in the documentation

Reply:
My bad, seems I missed or forgot that's not used anymore. Removing it.